ingest-mapper/src/test/scala/uk/gov/nationalarchives/ingestmapper/LambdaTest.scala

```diff
@@ -63,16 +63,21 @@ class LambdaTest extends AnyFlatSpec with MockitoSugar with BeforeAndAfterEach {
     dynamoRequestBodies.length should equal(1)
     val tableRequestItems = dynamoRequestBodies.head.RequestItems.test
 
-    tableRequestItems.length should equal(12)
+    tableRequestItems.length should equal(11)
     case class TestResponses(response: NetworkResponse, uuidIndices: List[Int], series: String)
```

Comment:
It's not really only Dynamo responses because of the indices and the series so I think TestResponses is generic enough. I've changed it to `TestResponses(dynamoResponse: DynamoResponse, uuidIndices: List[Int], series: String)` though.